### PR TITLE
Added support to add members in an organization

### LINKED ISSUE #1653: ADMIN REDESIGN: Add User/Member Capability
**Is your feature request related to a problem? Please describe.**

We need to add new features as part of the Admin Portal redesign.

**Describe the solution you'd like**

We need the ability for Admins to:

1. add a brand new user to the system and automatically make them members of their organizations.
1. make sure they are not adding preexisting users to the system, creating duplicates. (Make the user's email address the unique identifier.)
1. access this feature from the Organization People screen. (Refer to this issue)
    1. https://github.com/PalisadoesFoundation/talawa-admin/issues/1550

**Figma**

- https://www.figma.com/file/WksD4uIa5sQJMIk3pR4l0c/Talawa-Desi

**Describe alternatives you've considered**

- N/A

**Approach to be followed (optional)**

- N/A

**Additional context**

1. **_NOTE:_** This is going to require someone with experience with the repo, or proven long term experience with typescript, MERN, and / or graphQL in their GitHub profile
1. You will need to work closely with the person assigned this issue as the design elements overlap
    1. https://github.com/PalisadoesFoundation/talawa-admin/issues/1615

**Potential internship candidates**

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by disha1202**
**What kind of change does this PR introduce?**

Feature
**Issue Number:**

Fixes #1653 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/58461801/266b11a9-b83e-4746-b2c1-e1b8425b1f9a

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by disha1202** [comment]
@palisadoes 
I have added the support to add members to the organization. But right now only SUPER ADMINS  can perform this action
Screenshot 2024-03-12 at 6 34 46 PM

This needs to be fixed on the API side so that ADMINS of the organization can add members as well.

**Comment by palisadoes** [comment]
Please fix the failing tests

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Count number of lines
         run: |
           chmod +x ./.github/workflows/countline.py
-          ./.github/workflows/countline.py --lines 600 --exclude_files src/screens/LoginPage/LoginPage.tsx
+          ./.github/workflows/countline.py --lines 600 --exclude_files src/screens/LoginPage/LoginPage.tsx src/GraphQl/Queries/Queries.ts src/screens/OrganizationPeople/OrganizationPeople.tsx
```
[the remark this answers is not in the document]
Comment (palisadoes) [reply .github/workflows/pull-request.yml:41]: Yes. That was the aim of the line count test. This is the required approach

[the remark this answers is not in the document]
Comment (disha1202) [reply .github/workflows/pull-request.yml]: Thanks Updated!!

[the remark this answers is not in the document]
Comment (palisadoes) [reply .github/workflows/pull-request.yml]: You need to revert this file

**Comment by Cioppolo14** [comment]
@Kevoniat @Tajcore Can you review this PR?

**Review by palisadoes — APPROVED** [review]

**Pull request merged**